consensus/quorum/one-node-staked-vote.go

```diff
@@ -2,6 +2,7 @@ package quorum
 
 import (
 	"encoding/json"
+	errors2 "errors"
```

Comment:
no errors2, just find usages of `pkg/errors` , use that instead of builtin errors please